scripts/install-gphoto-python.sh

```diff
@@ -88,15 +88,52 @@ function clean_service() {
     fi
 }
 
+function create_ffmpeg_webcam_service() {
+    info "### Create Service for running CameraControl Daemon..."
+    cat >/etc/systemd/system/ffmpeg-webcam.service <<EOF
+[Unit]
+Description=ffmpeg webcam service
+
+[Service]
+Type=simple
+RemainAfterExit=no
+ExecStart=/usr/bin/python3 /var/www/html/api/cameracontrol.py --forceRecreateCam
```

Comment:
Path might be different, /var/www/html/photobooth/api/cameracontrol.py is also possible. Should  be checked and set as needed

Reply:
Sorry, another thing I've missed.

Reply:
Don't worry, that's what's a code review is made for =)

I've used Gerrit code review on Android development for a long time.